prow/ANNOUNCEMENTS.md

```diff
@@ -3,6 +3,11 @@
 ## New features
 
 New features added to each component:
+- *June 12, 2020* Added _bucket_ and _folder_ validation features to Deck to restrict artifact requests to only
```

Comment:
I think it would be good to do this by default

Reply:
Since this is a breaking change, we won't introduce it immediately. I will use a `*bool` for the "turn it on" flag so that we can detect whether it is explicitly set. I will also add warnings to the logs to indicate when the settings need to be updated.

Reply:
Done. (Warnings added, "in the future" comments added, etc.)